parlai/scripts/train_model.py

```diff
@@ -57,7 +57,7 @@ def setup_args(parser=None):
     train.add_argument('-sval', '--save-after-valid', type='bool',
                        default=False,
                        help='Saves the model to model_file.checkpoint after '
-                            'every validation (default True).')
+                            'every validation (default False).')
```

Comment:
You can also write `%(default)s` instead and argparse will insert the default.

Reply:
I' ll change that, thanks!